src/server/paths/root-router.node.spec.js

```diff
@@ -1,12 +1,10 @@
 import express from 'express';
-import path from 'path';
```

Comment:
MongoDB doesn't like us mocking `path`. It's probably not a good thing to mock generally.